src/operator/src/insert.rs

```diff
@@ -252,6 +266,8 @@ impl Inserter {
         .convert(requests)
         .await?;
 
+        // Fill impure default values in the request
+        let inserts = fill_reqs_with_impure_default(&table_infos, inserts)?;
         self.do_request(inserts, &ctx).await
```

Comment:
Can we move `fill_reqs_with_impure_default()` into `do_request()`? We can refactor `do_request()` to accept the `table_infos`.